portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -58,7 +49,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     Comment comment = new Comment(name,text);
-    comments.add(comment);
+    Entity com = new Entity("Comment");
```

Comment:
I may be missing something here, but is there a difference between using the `Comment comment` that has already been created with name/text and the `Entity com` that you're creating here?

Reply:
to be honest I'm not completely sure but that's how the walkthrough told us to do it
I think it has to do with saving it in Datastore so that when you refresh or restart the server it stays

Reply:
OK sounds good to me. Thanks for the clarification!